src/ring_theory/dedekind_domain/ideal.lean

```diff
@@ -211,22 +198,21 @@ open fractional_ideal
 variables {R A K}
 
 lemma is_dedekind_domain_inv_iff [algebra A K] [is_fraction_ring A K] :
-  is_dedekind_domain_inv A ↔
-    (∀ I ≠ (⊥ : fractional_ideal A⁰ K), I * I⁻¹ = 1) :=
+  is_dedekind_domain_inv A ↔ (∀ I ≠ (⊥ : fractional_ideal A⁰ K), I * I⁻¹ = 1) :=
 begin
-  let h := fractional_ideal.map_equiv (fraction_ring.alg_equiv A K),
+  let h := map_equiv (fraction_ring.alg_equiv A K),
   refine h.to_equiv.forall_congr (λ I, _),
   rw ← h.to_equiv.apply_eq_iff_eq,
   simp [is_dedekind_domain_inv, show ⇑h.to_equiv = h, from rfl],
 end
 
-lemma fractional_ideal.adjoin_integral_eq_one_of_is_unit [algebra A K] [is_fraction_ring A K]
+lemma adjoin_integral_eq_one_of_is_unit [algebra A K] [is_fraction_ring A K]
```

Comment:
Is this change intended?

Reply:
It's not, thanks! I just did a global search and replace and didn't notice this